### PR TITLE
Make our installer documentation on SSL setup more explicit

### PR DESCRIPTION
**Pull request by armchairlinguist**
This is a common area of confusion, so the more explicit we can be, the better for everyone.

Because of the recent changes in #498 I adjusted sectioning and also caught a few inconsistencies while I was at it.

### CODE REVIEW

File: content/source/docs/enterprise/private/preflight-installer.html.md [hunk 1]
```diff
@@ -12,12 +12,14 @@ Before installing the Private Terraform Enterprise software, you'll need to prep
...
 * TLS private key and certificate
   * The installer allows for using a certificate signed by a public or private CA. If you do not use a trusted certificate, your VCS provider will likely reject that certificate when sending webhooks. The key and X.509 certificate should both be PEM (base64) encoded.
-* License key (provided by HashiCorp)
+* License file (provided by HashiCorp)
 
 ~> **Note:** If you use a certificate issued by a private Certificate
    Authority, you must provide the certificate for that CA in the
    `Certificate Authority (CA) Bundle` section of the installation. This allows services
    running within PTFE to access each other properly.
+   See [Installation: Trusting SSL/TLS Certificates](./install-installer#trusting-ssl-tls-certificates)
```
Comment (nfagerlund) [inline content/source/docs/enterprise/private/preflight-installer.html.md]: OH WAIT, this is a broken link. Needs a `.html` in there! 

Once that's fixed, my 👍 is real. 😅

Comment (armchairlinguist) [reply content/source/docs/enterprise/private/preflight-installer.html.md]: Fixed!

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 2]
```diff
@@ -60,13 +60,30 @@ On the Console Settings page, there is a section for HTTP Proxy:
...
 ## Trusting SSL/TLS Certificates
 
-The installer has a section that allows multiple certificates to be specified as trusted.
-A collection of certificates for trusted issuers is known as a `Certificate Authority (CA) Bundle` and is
-used to allow PTFE to connect to services that use SSL/TLS certificates issued by private CAs.
+There are two primary areas for SSL configuration in the installer.
 
+### TLS Key & Cert
+
+The TLS Key & Cert field (found in the console settings after initial installation) should contain PTFE's own key and certificate, or key and certificate chain. A chain would be used in this field if the CA indicates a chain is required.
```
Comment (koikonom) [inline content/source/docs/enterprise/private/install-installer.html.md]:
```suggestion
The TLS Key & Cert field (found in the console settings after initial installation) should contain PTFE's own key and certificate, or key and certificate chain. A chain would be used in this field if the CA indicates an intermediate certificate is required as well.
```

File: content/source/docs/enterprise/private/install-installer.html.md [hunk 3]
```diff
@@ -60,13 +60,30 @@ On the Console Settings page, there is a section for HTTP Proxy:
...
+
+### Certificate Authority (CA) Bundle
+
+PTFE needs to be able to access all services that it integrates with, such as VCS providers.
```
Comment (koikonom) [inline content/source/docs/enterprise/private/install-installer.html.md]:
```suggestion
PTFE needs to be able to access all services that it integrates with, such as VCS providers or database servers.
```

File: content/source/docs/enterprise/private/migrate.html.md [hunk 4]
```diff
@@ -77,15 +77,32 @@ On the Console Settings page, there is a section for HTTP Proxy:
...
+## Trusting SSL/TLS Certificates
 
-The installer has a section that allows multiple certificates to be specified as trusted.
-A collection of certificates for trusted issuers are known as a `Certificate Authority (CA) Bundle` and are
-used to allow PTFE to connect to services that use SSL/TLS certificates issued by private CAs.
+There are two primary areas for SSL configuration in the installer.
 
+### TLS Key & Cert
+
+The TLS Key & Cert field (found in the console settings after initial installation) should contain PTFE's own key and certificate, or key and certificate chain. A chain would be used in this field if the CA indicates a chain is required.
```
Comment (koikonom) [inline content/source/docs/enterprise/private/migrate.html.md]: As above.

Comment (armchairlinguist) [reply content/source/docs/enterprise/private/migrate.html.md]:
```suggestion
The TLS Key & Cert field (found in the console settings after initial installation) should contain PTFE's own key and certificate, or key and certificate chain. A chain would be used in this field if the CA indicates an intermediate certificate is required as well.
```

**Pull request merged**